packages/subcommands/src/lib/structures/Command.ts

```diff
@@ -0,0 +1,30 @@
+import { Args, Command as BaseCommand, CommandOptions } from '@sapphire/framework';
+import type { PieceContext } from '@sapphire/pieces';
+import type { SubCommandOptionsResolvable } from '../utils/SubCommandsHandler';
+
+export abstract class Command<T = Args> extends BaseCommand<T> {
+	/**
+	 * Sub commands which this command have.
+	 * @since 0.0.1
```

Comment:
```suggestion
	 * @since 1.0.0
```
Change all other instances to this as well, since once it is released, it will no longer be 0.0.1. That's just a placeholder for lerna.